quickwit-config/resources/tests/server_config/server.toml

```diff
@@ -0,0 +1,13 @@
+metastore_uri = "postgres://username:password@host:port/db"
+
+[indexer]
+data_dir_path = "/opt/quickwit/data"
+
+[searcher]
+data_dir_path = "/opt/quickwit/data"
+peer_seeds = [ "quickwit-searcher-0.local", "quickwit-searcher-1.local" ]
+fast_field_cache_capacity = "1G"
+split_footer_cache_capacity = "500M"
+
+[storage]
+s3 = { region = "us-east-1", endpoint = "https://s3.us-east-1.amazonaws.com" }
```

Comment:
what is this s3 endpoint for?

Reply:
This is the URL of the AWS S3 API endpoint. Some accounts have custom endpoints (and the region is ignored).